tests/utils/verification/volumeproperties.go

```diff
@@ -35,7 +35,7 @@ import (
 // GetVMAttachedToVolUsingDockerCli returns attached to vm field of volume using docker cli
 // TODO: make this private member after finishing refactoring of volmprop_test.go and remove this TODO
 func GetVMAttachedToVolUsingDockerCli(volName string, hostname string) string {
-	cmd := dockercli.InspectVolume + " --format '{{index .Status \"attached to VM\"}}' " + volName
+	cmd := dockercli.InspectVolume + " --format \"{{index .Status \\\"attached to VM\\\"}}\" " + volName
```

Comment:
That's because the Windows [`ssh` utility](https://github.com/vmware/docker-volume-vsphere/blob/96e104ef6ae09ae1240eefb5c02170a2b805f15a/tests/utils/ssh/ssh_win.go#L28) wraps the command in single quotes.